src/components/IconLibrary/IconLibrary.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Search, Loading } from 'carbon-components-react';
 import IconEmptyState from '../IconEmptyState';
 
-const sizes = ['16', '32', 'Glyph'];
+const sizes = ['16', '20', '24', '32', 'Glyph'];
```

Comment:
`const sizes = ['16', '20', '24', '32'];`
This would remove the Glyph section all together. Is that what you want? @chrisconnors-ibm